Dockerfile

```diff
@@ -1,16 +1,20 @@
-FROM node:10
+FROM node:10-slim
 
-RUN mkdir /app
 WORKDIR /app
 
-COPY package.json ./
-COPY yarn.lock ./
+ARG rctf_name
+ENV RCTF_NAME=${rctf_name}
 
-ENV NODE_ENV production
+COPY package.json yarn.lock ./
 RUN yarn
```

Comment:
Could we combine these lines into ```RUN yarn && yarn build && rm -r node_modules```. I think it makes it clearer what the intent is, install dev dependencies and remove node_modules